txpipe/utils/calibration_tools.py

```diff
@@ -164,10 +164,10 @@ def __init__(self, selector, delta_gamma):
             The difference in applied g between 1p and 1m metacal variants
```

Comment:
It's probably easier to look at this class (`ParallelCalibratorMetacal`) in its entirety rather than the diff.